src/main/java/seedu/address/model/profile/Name.java

```diff
@@ -56,4 +56,9 @@ public int hashCode() {
         return fullName.hashCode();
     }
 
+    @Override
+    public int compareTo(Name other) {
+        return this.fullName.compareTo(other.fullName);
```

Comment:
Perhaps doing a case-insensitive comparison would be better, right now lowercase characters goes right to the bottom.
```suggestion
        return this.fullName.toLowerCase().compareTo(other.fullName.toLowerCase());
```